packages/react-card/src/components/Card/renderCard.tsx

```diff
@@ -1,17 +1,15 @@
 import * as React from 'react';
 import { getSlotsCompat } from '@fluentui/react-utilities';
-import { cardShorthandProps } from './useCard';
 import type { CardState } from './Card.types';
 
 /**
  * Render the final JSX of Card
  */
 export const renderCard = (state: CardState) => {
-  const { slots, slotProps } = getSlotsCompat(state, cardShorthandProps);
+  const { slots, slotProps } = getSlotsCompat(state);
 
   return (
-    <slots.root {...slotProps.root}>
-      {/* TODO Add additional slots in the appropriate place */}
+    <slots.root role="group" {...slotProps.root}>
```

Comment:
the `role` should be defined and defaulted in the component state

```suggestion
    <slots.root {...slotProps.root}>
```